device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientOptions.java

```diff
@@ -3,14 +3,20 @@
 
 package com.microsoft.azure.sdk.iot.device;
 
+import com.microsoft.azure.sdk.iot.deps.convention.DefaultPayloadConvention;
+import com.microsoft.azure.sdk.iot.deps.convention.PayloadConvention;
+import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 import javax.net.ssl.SSLContext;
 
 /**
  * Options that allow configuration of the device client instance during initialization.
  */
+@Builder
```

Comment:
@timtay-microsoft I don't believe this will break anything, do you foresee any issues with making this a builder class as well?

Reply:
Since you added the noArgsConstructor annotation as well, you should be fine to do this, yeah